modules/mapbox/src/deck-utils.ts

```diff
@@ -300,7 +300,12 @@ type MaplibreRenderParameters = {
 
 function getViewport(deck: Deck, map: Map, renderParameters?: unknown): Viewport {
   const viewState = getViewState(map);
-  const view = getDefaultView(map);
+  const {
+    props: {views}
+  } = deck;
+  const view =
```

Comment:
Is it time to define a `getView()` function and place this logic there rather than hide in here?